<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>1815</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>1024</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1040</width>
    <height>1840</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SZ Tombola</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>1024</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>1040</width>
     <height>1840</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1121</width>
      <height>1741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Gl√ºcksrad</string>
     </attribute>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>491</width>
        <height>301</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Images.qrc">:/images/SZ-Ybbs-Logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLCDNumber" name="resultLCD">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>1011</width>
        <height>651</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <pointsize>128</pointsize>
        <bold>false</bold>
       </font>
      </property>
      <property name="digitCount">
       <number>4</number>
      </property>
     </widget>
     <widget class="QPushButton" name="startStopButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>1560</y>
        <width>1011</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="timeLCD">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>1000</y>
        <width>501</width>
        <height>541</height>
       </rect>
      </property>
      <property name="digitCount">
       <number>2</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Filled</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>10</y>
        <width>521</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>94</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Tombola </string>
      </property>
     </widget>
     <widget class="QLabel" name="yearLabel">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>150</y>
        <width>491</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>94</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>2023</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Zahlenbereich</string>
     </attribute>
     <widget class="QSpinBox" name="rangeSB">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>271</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="maximum">
       <number>99999</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>71</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>1 - </string>
      </property>
     </widget>
     <widget class="QSpinBox" name="timeSB">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>400</y>
        <width>141</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>481</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Zahlenbereich: </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>300</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Laufzeit:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>400</y>
        <width>291</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sekunden</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="yearSpinBox">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>610</y>
        <width>261</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="minimum">
       <number>2022</number>
      </property>
      <property name="maximum">
       <number>2100</number>
      </property>
      <property name="value">
       <number>2023</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>530</y>
        <width>391</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Jahr:</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Images.qrc"/>
 </resources>
 <connections/>
</ui>
